modules/oc-mirror-enclave-support.adoc

```diff
@@ -0,0 +1,141 @@
+
+.Phase 2 - Mirroring to the enclave
+
+Once all mirroring content has been transferred to the enterprise's central registry (enterprise-registry.in in the example), the users might be interested in mirroring {product-title} content from that internal registry to one or more enclaves.
+
+In order to do so, the same or a new `imageSetConfig`, describing the content that needs to be mirrored to the enclave is prepared.
+
+.Example isc-enclave.yaml
+[source,yaml]
+----
+apiVersion: mirror.openshift.io/v2alpha1
+kind: ImageSetConfiguration
+mirror:
+  platform:
+    architectures:
+      - "amd64"
+    channels:
+      - name: stable-4.15
+        minVersion: 4.15.2
+        maxVersion: 4.15.2
+----
+
+oc-mirror needs to be run on a machine with access to the disconnected registry, that is, with reference to our example: in the air gap environment, where enterprise-registry.in is accessible.
+
+. Updating the `registries.conf`
++
+Before running oc-mirror in the air gap environment, the user needs to set up the `registries.conf` file. This is illustrated by arrow 4 in the diagram.
++
+The link:https://github.com/containers/image/blob/main/docs/containers-registries.conf.5.md#example[file specification] suggests to store the file under `$HOME/.config/containers/registries.conf`, otherwise `/etc/containers/registries.conf`.
++
+The TOML format of the file is also described in the above specification.
++
+.Example `registries.conf`
+[source,toml]
+----
+[[registry]]
+location="registry.redhat.io"
+[[registry.mirror]]
+location="<enterprise-registry.in>"
+
+[[registry]]
+location="quay.io"
+[[registry.mirror]]
+location="<enterprise-registry.in>"
+----
+
+.. Update Graph URL
++
+If you are using `graph:true`, oc-mirror will attempt to reach the cincinnati API endpoint. Since this environment is air gapped, make sure you export environment variable `UPDATE_URL_OVERRIDE` to reference the URL for the OSUS (OpenShift UpdateService), like so:
++
+[source,terminal]
+----
+$ export UPDATE_URL_OVERRIDE=https://<osus.enterprise.in>/graph
+----
++
+For more information on how to setup OSUS on an OpenShift cluster, please refer to the link:https://docs.openshift.com/container-platform/4.14/updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.html[documentation].
+
+. Generating a mirror archive from the enterprise registry for the enclave.
++
+In order to prepare an archive for the enclave1, as illustrated as arrow 5 in the diagram, the user executes oc-mirror in the enterprise airgap environment, using the `imageSetConfiguration` specific for that enclave. This ensures that only images needed by that enclave are mirrored:
```

Comment:
arrow 5 in the diagram ?

Reply:
I have removed the diagram instances, since we are not sure if we will get the diagram done. SO I will be removing all the wordings related to diagram.